### ISSUE TITLE
Could you bump standard to the latest version (5.3.1) please?

### ISSUE THREAD

**Issue by michaelgilley** [ricardofbarros/linter-js-standard#65]
Thanks!

Makes a difference:

```js
// Standard 5.2.1 says this is all clean
// Standard 5.3.1 reports correctly 'Unexpected whitespace before semicolon'
for (var _y = moment().format('YYYY'); _y >= 1924 ; --_y) {
  su.yearOptions.push({year: _y, abbrev: _y})
}
```

**Comment by kilianc** [comment]
+1

**Comment by tonyghita** [comment]
There is a pull-request out for upgrading standard to the latest: https://github.com/ricardofbarros/linter-js-standard/pull/89

**Comment by ricardofbarros** [comment]
Hello guys, @devTristan [PR](https://github.com/ricardofbarros/linter-js-standard/pull/101) fixed this. From now on it will try to use the local standard (project's node_modules/) instead of the bin inside this package. Many thanks to @devTristan
